### ISSUE TITLE
Preview field selecting text with shift and space bar bug

### ISSUE THREAD

**Issue by barackenkumpel** [florisboard/florisboard#1938]
#### Short description

When in preview field in settings:
- the "selecting mode" is escaped and entered shortly when beginning with dragging the space bar
- but when beeing allready dragging and then holding shift everything is fine

~\+ maybe it would be better if the keyboard doesn't enter caps lock after selecting text this way~ (#1932)

##### Screenrecords

https://user-images.githubusercontent.com/93818968/177522126-f528accf-faed-42bd-a047-75b381a93d1f.mp4

https://user-images.githubusercontent.com/93818968/177522147-12023999-867e-4927-99b7-e7f36935849d.mp4

#### Steps to reproduce
1. Set space bar swipe gestures to move cursor
2. In keyboard hold down Shift and drag the space bar

#### Environment information
- FlorisBoard Version: 0.3.16
- Install Source: F-Droid
- Device: Samsung (S20 FE)
- Android: 12 One UI (modified)

**Comment by patrickgold** [comment]
Could you please try this out in any other text field besides the preview field? Sadly the preview field itself is not the stablest thing on earth right now (due to issues in the UI library I use) and I suspect that this is caused by it.

**Comment by wiltRainbow** [comment]
You're right, other text fields aren't affected. I wouldn't have ever thought about this could be a preview field specific thing.

**Comment by barackenkumpel** [comment]
You're right, other text fields aren't affected. I wouldn't have ever thought about this could be a preview field specific thing. I'll edit the issue

**Comment by lm41** [comment]
This issue has been open for a long period of time. Does it still occur in the latest version (from my testing not)?
